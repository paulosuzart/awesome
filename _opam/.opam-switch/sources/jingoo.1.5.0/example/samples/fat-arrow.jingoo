((i) => i + 1)(1) = {{ ((i) => i + 1)(1) }}
(i => i + 1)(-1) = {{ (i => i + 1)(-1) }}
((a, b) => a + b)(1, 2) = {{ ((a, b) => a + b)(1, 2) }}
(() => 'foo')() = {{ (() => 'foo')() }}
