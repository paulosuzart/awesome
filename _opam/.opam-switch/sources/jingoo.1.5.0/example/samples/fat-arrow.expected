((i) => i + 1)(1) = 2
(i => i + 1)(-1) = 0
((a, b) => a + b)(1, 2) = 3
(() => 'foo')() = foo
